qa/setup_packages.py

```diff
@@ -417,11 +417,13 @@ def get_pyvers_name(self, url, cuda_version):
                               PckgVer("1.15.5+nv21.06", python_min_ver="3.8", python_max_ver="3.8", alias="nvidia-tensorflow")]
                         }),
                 CudaPackageExtraIndex("torch",
-                        { "100" : ["1.4.0"],
-                          "110" : ["1.4.0"] }, extra_index="https://download.pytorch.org/whl/cu{cuda_v}/"),
+                        { "100" : ["1.7.0"],
+                          "101" : ["1.7.0"],
+                          "110" : ["1.7.0"] }, extra_index="https://download.pytorch.org/whl/cu{cuda_v}/"),
                 CudaPackageExtraIndex("torchvision",
-                        { "100" : ["0.5.0"],
-                          "110" : ["0.5.0"] }, extra_index="https://download.pytorch.org/whl/cu{cuda_v}/"),
+                        { "100" : ["0.8.1"],
+                          "101" : ["0.8.1"],
```

Comment:
Is this CUDA 10.1 version? Why not 10.2?

Reply:
We could but we don't have 10.2 in our base test image. Only 10.0 and 10.1. I don't think it is worth adding it just because of it.